src/utils.ts

```diff
@@ -427,10 +437,7 @@ export function applyPathMapping(
         const wildcardValue = overridePatternMatches[1];
         let mappedPath = rightPattern.replace(/\*/g, wildcardValue);
         mappedPath = debugCore.utils.properJoin(mappedPath); // Fix any ..'s
-        mappedPath = mappedPath.replace(
-            "${workspaceFolder}",
-            vscode.workspace.workspaceFolders ? vscode.workspace.workspaceFolders[0].uri.toString() : "" || "");
-
+        mappedPath = replaceWorkSpaceFolderPlaceholder(mappedPath);

```

Comment:
There is a new behavior here after the change.  If `vscode.workspace.workspaceFolders` is `null` or `undefined`, in this branch, the returned path will still contain `${workspaceFolder}` in the path.  Before this change, the mapped path would remove the `${workspaceFolder}`.

Is this intentional?

Reply:
Yes, this is part of the  bug fix. In this specific case you have:
`await transformer.fixSource(localSource)` a couple of lines below which should take care of correctly mapping sourcePath. I also enabled a test for this scenario (turned sourceMaps true).